tests/PHPStan/Analyser/data/specifiedTypesUsingIsFunctions.php

```diff
@@ -41,7 +41,8 @@ function () {
 		throw new \Exception();
 	}
 
-	if (!is_resource($resource)) {
+	$resource = fopen('php://memory', 'r');
+	if ($resource === false) {
```

Comment:
Not sure why the change is here. Usually changing existing (and especially this old) tests is a red flag. Did we break something?

Reply:
Reverted, not needed.